tests/ui.js

```diff
@@ -32,12 +32,16 @@ function tapLogger(type, file, err) {
 process.chdir(__dirname);
 const tester = require('screenshot-tester')(
 	{outDir: '_snapshots', accuracy: 500, logger: tapLogger});
+const testerRaw = require('screenshot-tester')(
```

Comment:
What is this line for?

Reply:
```
tester('../example/gallery/start.js', 'Control Gallery', {raw: true})
```
would also work.

This is what raw does:
```js
if (raw) {
	proc = spawn(file);
} else {
	proc = spawn("node", [file]);
}
```

Reply:
> tester('npm start', 'Control Gallery', {raw: true})

Nice, i'll use this

Reply:
> Nice, i'll use this

I'm updating screenshot-tester at the moment to support passing arguments. Currently it tries to execute the file "npm start".

Reply:
Actually, I'm using a start.js file I wrote in the example/gallery folder.
I'll also have to run `npm install` before, because on CI that example load also other packages.

Reply:
Should missing reference images fail the test or skip as with core-api (see other PR)?